### ISSUE TITLE
Version 2.4.8.2 Doesn't seem to contain the changes

### ISSUE THREAD

**Issue by mdius** [concretecms-community-store/community_store#703]
It appears that Version 2.4.8.2 is the same as the previous 2.4.7.8.

**Comment by mdius** [comment]
Yes, the version it shows is 2.4.8.2, but when I look at the changed files, they are the same as the older version.

Like the files:
blocks\community_product_filter/controller.php
blocks\community_product_filter/db.xml
blocks\community_product_filter/edit.php
blocks\community_product_filter/view.php

All the files I looked at so far were the same as the previous version. I re-downloaded the new release and it is the same, was hoping maybe I did something when unzipping it.
I am assuming this release was to include the [Add product list and filter block filtering by product type](https://github.com/concretecms-community-store/community_store/commit/ea52f6e5e0b29d860b7ceff7ce16ae96fd4f74cd)?

**Comment by Mesuva** [comment]
No, the product list and filter block filtering by product type actually isn't quite in the 2.4.8.2 release - that was added shortly after I made the release. It'll be added soon in 2.4.8.3.

Would that perhaps then explain what you're seeing?

**Comment by mdius** [comment]
Oh, I see what I did, I assumed since they were released 3 days ago (at the time the update and the file change notes in GitHub were 3 days).
So, yes, that would explain it. I am very sorry about that. I did a lot of customizing and when I get an update I go through your change log and compare.

**Comment by Mesuva** [comment]
No worries at all, I'd rather you ask - easy for me to mix up files, I've done this before!
